packages/abi-coder/src/types/JsonAbi.ts

```diff
@@ -2,15 +2,12 @@
  * Types for Fuel JSON ABI Format as defined on:
  * https://github.com/FuelLabs/fuel-specs/blob/master/src/abi/json-abi-format.md
  */
-
-import type { EncodingVersion } from '../utils/constants';
-
 export interface JsonAbi {
   readonly types: readonly JsonAbiType[];
   readonly loggedTypes: readonly JsonAbiLoggedType[];
   readonly functions: readonly JsonAbiFunction[];
   readonly configurables: readonly JsonAbiConfigurable[];
-  readonly encoding?: EncodingVersion;
+  readonly encoding?: string;
```

Comment:
Can you explain why we'd go from `EncodingVersion` to `string` when this field is optional to begin with so the user doesn't have to type it at all?